salt/orchestrate/aws/cloud_profiles/elasticsearch.conf

```diff
@@ -6,6 +6,9 @@ elasticsearch:
   ssh_interface: private_ips
   block_device_mappings:
     - DeviceName: /dev/xvda
+      Ebs.VolumeSize: 20
+      Ebs.VolumeType: gp2
+    - DeviceName: /dev/xvdb
       Ebs.VolumeSize: 400
```

Comment:
I left it that size for when we are deploying things like our logging cluster. For other instances we can override that value in the map file.